src/medCore/gui/database/medDatabaseView.cpp

```diff
@@ -559,13 +559,23 @@ void medDatabaseView::onEditRequested(void)
         QList<QVariant> attributes = item->attributes();
         QList<QVariant> values = item->values();
         QList<QString> labels;
-        
+
+        // The last attribute is ThumbnailPath. Users are not allowed to change it.
+        attributes.removeLast();
+        values.removeLast();
```

Comment:
I don't think you should rely on it being last as that is just a secondary effect of the sorting algorithm. You should search the lists and remove the one corresponding the the thumbnail path metadata key.